routers/repo/view.go

```diff
@@ -195,6 +195,7 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 		ctx.Data["ReadmeExist"] = readmeExist
 		if markup.Type(blob.Name()) != "" {
 			ctx.Data["IsRenderedHTML"] = true
```

Comment:
`ctx.Data["IsRenderedHTML"] = true` can be removed (check other places, same as `markup.Type`) if `ctx.Data["IsMarkup"] = true` is set, then you can remove nested `if` from template  and use `else if`.

Reply:
Where should I remove the `if` in template? (Could you provide a Line?)

Reply:
https://github.com/go-gitea/gitea/pull/2721/files#diff-2ec327528ed2be8dd3c2201ef8e6a42cR40